Bottom: a880ced2da4d3738bdf0b76d4f8ad144557016b8
Top:    f4de70681a1f60a6eade68e9c27c794f529b77a5
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-11 13:26:07 -0700

Refresh of SBInstruction

---
diff --git a/lldb/scripts/interface/SBInstruction.i b/lldb/scripts/interface/SBInstruction.i
index a46d845ff6a..4586978733f 100644
--- a/lldb/scripts/interface/SBInstruction.i
+++ b/lldb/scripts/interface/SBInstruction.i
@@ -60,7 +60,7 @@ public:
     Print (lldb::SBFile out);
 
     void
-    Print (lldb::FileSP &BORROWED);
+    Print (lldb::FileSP BORROWED);
 
     bool
     GetDescription (lldb::SBStream &description);
